### PR TITLE
Add table aws_vpc_verified_access_instance closes #1467

### LINKED ISSUE #1467: Add table aws_vpc_verified_access_instance
**References**
https://docs.aws.amazon.com/AWSEC2/latest/APIReference/API_DescribeVerifiedAccessInstances.html

### PR DESCRIPTION
**Pull request by ParthaI**
# Integration test logs
<details>
  <summary>Logs</summary>

```
N/A
```
</details>

# Example query results
<details>
  <summary>Results</summary>

```
> select
  verified_access_instance_id,
  creation_time,
  description,
  last_updated_time,
  verified_access_trust_providers
from
  aws_vpc_verified_access_instance;
+-----------------------------+---------------------------+-------------+---------------------------+-----------------------------------------------------------------------------------------------------
| verified_access_instance_id | creation_time             | description | last_updated_time         | verified_access_trust_providers                                                                     
+-----------------------------+---------------------------+-------------+---------------------------+-----------------------------------------------------------------------------------------------------
| vai-011ed46e43d2d1b1e       | 2022-12-05T14:00:40+05:30 | test        | 2022-12-05T14:00:41+05:30 | [{"Description":"test","DeviceTrustProviderType":"","TrustProviderType":"user","UserTrustProviderTyp
+-----------------------------+---------------------------+-------------+---------------------------+-----------------------------------------------------------------------------------------------------

Time: 4.4s. Rows fetched: 1. Hydrate calls: 0.
> select
  verified_access_instance_id,
  creation_time,
  description,
  last_updated_time
from
  aws_vpc_verified_access_instance
where
  creation_time >= now() - interval '30' day;
+-----------------------------+---------------------------+-------------+---------------------------+
| verified_access_instance_id | creation_time             | description | last_updated_time         |
+-----------------------------+---------------------------+-------------+---------------------------+
| vai-011ed46e43d2d1b1e       | 2022-12-05T14:00:40+05:30 | test        | 2022-12-05T14:00:41+05:30 |
+-----------------------------+---------------------------+-------------+---------------------------+

Time: 1.6s. Rows fetched: 1. Hydrate calls: 0.
> select
  i.verified_access_instance_id,
  i.creation_time,
  p ->> 'Description' as trust_provider_description,
  p ->> 'TrustProviderType' as trust_provider_type,
  p ->> 'UserTrustProviderType' as user_trust_provider_type,
  p ->> 'DeviceTrustProviderType' as device_trust_provider_type,
  p ->> 'VerifiedAccessTrustProviderId' as verified_access_trust_provider_id,
  t.policy_reference_name as trust_access_policy_reference_name
from
  aws_vpc_verified_access_instance as i,
  aws_vpc_verified_access_trust_provider as t,
  jsonb_array_elements(verified_access_trust_providers) as p
where
  p ->> 'VerifiedAccessTrustProviderId' = t.verified_access_trust_provider_id;
+-----------------------------+---------------------------+----------------------------+---------------------+--------------------------+----------------------------+-----------------------------------+
| verified_access_instance_id | creation_time             | trust_provider_description | trust_provider_type | user_trust_provider_type | device_trust_provider_type | verified_access_trust_provider_id |
+-----------------------------+---------------------------+----------------------------+---------------------+--------------------------+----------------------------+-----------------------------------+
| vai-011ed46e43d2d1b1e       | 2022-12-05T14:00:40+05:30 | test                       | user                | iam-identity-center      |                            | vatp-0f71337c0114b9e3e            |
+-----------------------------+---------------------------+----------------------------+---------------------+--------------------------+----------------------------+-----------------------------------+
```
</details>

### CODE REVIEW

**Comment by ParthaI** [comment]
Go lang lint suppose to fail until the [PR](https://github.com/turbot/steampipe-plugin-aws/pull/1482) gets merged into the main.

**Comment by misraved** [comment]
@ParthaI could you please take a look at the lint failure?

**Comment by ParthaI** [comment]
@misraved Is is supposed to fail. I have added a comment in this PR about why it will fail. There is a dependency on this [PR](https://github.com/turbot/steampipe-plugin-aws/pull/1482) which need to be merged first before merging this PR.

File: aws/table_aws_vpc_verified_access_instance.go [hunk 1]
```diff
@@ -0,0 +1,170 @@
...
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "last_updated_time",
+				Description: "The last updated time.",
+				Type:        proto.ColumnType_TIMESTAMP,
+			},
+			{
+				Name:        "verified_access_trust_providers",
+				Description: "The IDs of the AWS Verified Access instance.",
```
Comment (misraved) [inline aws/table_aws_vpc_verified_access_instance.go:53]:
```suggestion
				Description: "The IDs of the AWS Verified Access trusted providers.",
```

File: docs/tables/aws_vpc_verified_access_instance.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+  verified_access_instance_id,
+  creation_time,
+  description,
+  last_updated_time,
+  verified_access_trust_providers
+from
+  aws_vpc_verified_access_instance;
+```
+
+### List trusted providers older than 30 days
```
Comment (misraved) [inline docs/tables/aws_vpc_verified_access_instance.md:20]:
```suggestion
### List VPC access verified instances older than 30 days
```

**Review by misraved — COMMENTED** [review]
Please take a look at the review comments!!

**Review by misraved — APPROVED** [review]

**Pull request merged**